open-api/rest-catalog-open-api.yaml

```diff
@@ -1581,6 +1607,17 @@ components:
         type: string
       example: [ "accounting", "tax" ]
 
+    PageToken:
+      description:
+        An opaque token which allows clients to make use of pagination for a list API (e.g. ListTables). 
+        Clients will initiate the first paginated request by sending an empty `pageToken` e.g. `GET /tables?pageToken` or `GET /tables?pageToken=`
+        signaling to the service that the response should be paginated.
+
+        For servers that support pagination, they will recognize `pageToken` and return a `next-page-token` in response if there are more results available.
```

Comment:
I still think the wording can be improved in this section to avoid using `they` to refer to servers. I did a previous suggestion which avoids using `they`

Reply:
additionally it's confusing to use `pageToken` and `next-page-token` (vs `nextPageToken`), so which one is correct here?

Reply:
+ 1 for using next-page-token